### DISCUSSION TITLE
Support for additional video codecs

### DISCUSSION THREAD

**Discussion by phly95** [Floorp-Projects/Floorp#763] [Feature Request]
For sites like Animebook and asbplayer, the lack of wide codec support means that many mkv files that use ac3 audio or h265 video cannot be opened by the browser. Currently, the only way to open all video files is to use custom builds of Chromium or Microsoft Edge.

**Comment by Telmesomething** [comment]
VLC media player will play mkv file along with H265 video. Download from https://www.videolan.org/

Alternatively, you could convert them to MP4 with the same player.

**Reply by phly95** [reply]
Animebook is a browser app that allows you to use dictionary extensions with it for language learning purposes. While other apps like Memento exist as well, it is nice having all the existing browser extensions to integrate with videos in this way.

**Comment by kunkanader** [comment]
This is actually a very good request. Firefox suffers a lot from the lack of HEVC support. Using external players for this task is obviously a worse and much less convenient solution. Unfortunately it requires payment for native implementation. But maybe someone could make an unofficial plugin?

**Comment by ioc2e3** [comment]
>  ...。目前，開啟所有影片檔案的唯一方法是使用 Chromium 或 Microsoft Edge 的自訂版本。

Really , very sad thing .

.
